archunit/src/test/java/com/tngtech/archunit/core/importer/ClassFileImporterSlowTest.java

```diff
@@ -35,6 +35,13 @@
 
 @Category(Slow.class)
 public class ClassFileImporterSlowTest {
+    /**
+     * Importing the full classpath may give thousands of classes in {@link sun} and (especially for Java 8) {@link com.sun} packages.
+     * Importing those would increase the memory footprint of the test tremendously, but not give an additional benefit for the test,
+     * so they are excluded for convenience.
+     */
+    private static final ImportOption EXCLUDE_SUN_PACKAGES = l -> !l.contains("/sun/");
```

Comment:
What I still don't understand here, what has changed? The rt.jar archive of JDK 8 has been imported for many years, right? Do you know if the memory footprint of ArchUnit was simply smaller a couple of years ago? And if yes by how much? 🤔

Reply:
In any case, since it always causes problems when we import `rt.jar` for JDK 8, I'd add this (and the comment) as part of `importJavaBaseOrRtAndJUnitJarAndFilesOnTheClasspath()` I think 🤔  Then you also wouldn't have to repeat it in every place where that method is used?

Reply:
> I'd add this (and the comment) as part of `importJavaBaseOrRtAndJUnitJarAndFilesOnTheClasspath()`

There's also `importJavaBase()`, but it's certainly a good idea to keep the workaround down with those implementation details. 👍 I'll push an update.